src/encoded/types/dataset.py

```diff
@@ -1386,6 +1386,86 @@ class FunctionalCharacterizationSeries(Series):
         'elements_references.files',
     ]
 
+    @calculated_property(condition='related_datasets', schema={
+        "title": "Examined loci",
+        "type": "array",
+        "items": {
+            "type": "object",
+        },
+        "notSubmittable": True,
+    })
+    def examined_loci(self, request, related_datasets=None):
+        if related_datasets is not None:
```

Comment:
Looks like `condition` is related_datasets so shouldn't need this.

Reply:
updated